docs/man/docker-cp.1.md

```diff
@@ -24,7 +57,18 @@ the exited container to the current dir on the host:
 
     # docker cp c071f3c3ee81:setup.sh .
 
+A configuration file is copied into a stopped container and is restarted:
```

Comment:
and the container is restarted... 

Otherwise is sounds like you're restarting the file. :)